packages/nimble-components/src/rich-text-editor/specs/README.md

```diff
@@ -110,7 +110,10 @@ _Props/Attrs_
 -   `isEmpty` - is a read-only property that indicates whether the editor is empty or not. This will be achieved through Tiptap's
     [isEmpty](https://tiptap.dev/api/editor#is-empty) API. The component and the Angular directive will have a getter method
     that can be used to bind it in the Angular application.
--   `fitToContent` - is a boolean attribute allows the text area to expand vertically to fit the content.
+-   `fit-to-content` - is a boolean attribute allows the text area to expand vertically to fit the content.
```

Comment:
If you're changing `fitToContent` to snake case would you want to do the same for `isEmpty`?

Reply:
Yea should consistently show as either the property (camelCase) or attribute (snake-case) form in the spec. Most specs use the attribute case. Make sure to do a quick search and align when in spec comments when changed.

Reply:
Good change! 👍